modules/nvidia-container-toolkit.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, ... }:
+with builtins; with lib; {
```

```suggestion
{
```
we shouldn't add more of those